jupyterhub/auth.py

```diff
@@ -88,6 +88,18 @@ class Authenticator(LoggingConfigurable):
         """
     ).tag(config=True)
 
+    blacklist = Set(
+        help="""
+        Blacklist of usernames that are not allowed to log in.
+
+        Use this with supported authenticators to restrict which users can not log in. This is an
+        additional blacklist that further restricts users, beyond whatever restrictions the
+        authenticator has in place.
+
+        If empty, does not perform any additional restriction.
+        """
```

Comment:
Can you add 

    .. versionadded: 0.9

to the end of the docstring here and in the new method ?